packages/upload-media/package.json

```diff
@@ -1,7 +1,6 @@
 {
 	"name": "@wordpress/upload-media",
 	"version": "1.0.0-prerelease",
```

Comment:
Let's change the version to `0.0.1`, that's a semver way to say that the package is unstable.